spec/features/cdr/cdr_history/cdr_filter_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe 'Cdrs index page', type: :feature, js: true do
+  subject do
+    within_filters { click_submit('Filter') }
+  end
+
+  include_context :login_as_admin
+  let!(:cdrs) { create_list(:cdr, 2) }
+
+  before do
+    Cdr::Cdr.delete_all
+    visit cdrs_path
+  end
+
+  describe 'filtering', js: true do
```

Comment:
you can squash it with upper describe and remove one level of block nesting

```ruby
RSpec.describe 'Cdrs index page filtering', js: true do
  # ...
end
```

also parent context/describe meta options are propagated to inner contexts and tests, so no need for inside context/describe which already has it `js: true`